aten/src/ATen/jiterator_macros.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
```

Comment:
Moved to a new file as `jit_macros.h` includes `CUDAConfig.h` which is only available in CUDA build.